### ISSUE TITLE
image contains unnecessary files after compilation

### ISSUE THREAD

**Issue by nils-wisiol** [desec-io/desec-static#8]
remove to save space and bandwidth for deployment

**Comment by peterthomassen** [comment]
This line https://github.com/desec-io/desec-static/blob/49d772bec5ea17746ef336cc0908f0a328702dcb/Dockerfile#L25 deletes `/usr/src/app` when everything is done, so any temporary files are already deleted.

Which are the unnecessary files in the image? Some npm/grunt/bower stuff outside `/usr/src/app`?

**Comment by nils-wisiol** [comment]
The image is considerably larger than before (went up from 200M to 700M), maybe due to root-installed bower components?

Here are all 3rd-level folders before #4 that contain more than 10M:

    42M	/usr/lib/x86_64-linux-gnu
    33M	/usr/share/locale
    19M	/usr/share/perl
    14M	/usr/share/doc
    11M	/usr/share/nginx

Here are all 3rd-level folder with more than 10M after #4:

    96M	/usr/lib/node_modules
    57M	/usr/lib/x86_64-linux-gnu
    35M	/usr/share/locale
    33M	/usr/share/doc
    27M	/root/.npm/registry.npmjs.org
    20M	/usr/lib/python2.7
    19M	/usr/share/perl
    15M	/usr/lib/git-core
    11M	/usr/share/nginx
    11M	/usr/lib/ruby

It appears that not much space can be saved without using `apt-get` again. I think this is not desirable.

I will close this as won't fix, please reopen with this needs further processing.

Command used to create the listings: `docker run -ti $IMAGE_ID du -xh --max-depth 2 / | egrep '^[0-9]+M.*\/.*\/.*' | sort -rh`
